org.ghost.desktop.json

```diff
@@ -0,0 +1,40 @@
+{
+  "app-id": "org.ghost.desktop",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.gnome.Sdk",
+  "command": "Ghost",
+  "rename-desktop-file": "Ghost.desktop",
+  "rename-icon": "Ghost",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=host",
+    "--share=network",
+    "--device=dri",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [
+    {
+      "name": "ghost",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar -x ghost.deb",
+        "tar -xf data.tar.xz",
+        "cp -r usr/* /app",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "cp /app/share/pixmaps/Ghost.png /app/share/icons/hicolor/512x512/apps/Ghost.png"
```

Comment:
You could just copy the icon with the correct name here and remove the rename-icon

Reply:
Ok, I will do this.

Reply:
I tried this, but when doing so my distro somehow isn't showing the icon.

Reply:
Because you should patch the desktop file too. Using something like this https://github.com/flathub/com.github.z.Cumulonimbus/blob/785266a8ec0c2ebb73eea05571518238e193eb0c/com.github.z.Cumulonimbus.json#L33

Reply:
Fixed that